pom.xml

```diff
@@ -4,14 +4,18 @@
   <parent>
     <groupId>org.jenkins-ci.plugins</groupId>
     <artifactId>plugin</artifactId>
-    <version>1.532.3</version>
+    <version>2.31-SNAPSHOT</version>
   </parent>
   <artifactId>validating-string-parameter</artifactId>
   <packaging>hpi</packaging>
   <name>Validating String Parameter Plugin</name>
-  <version>2.4-SNAPSHOT</version>
+  <version>2.5-SNAPSHOT</version>
   <url>http://wiki.jenkins-ci.org/display/JENKINS/Validating+String+Parameter+Plugin</url>
 
+  <properties>
+    <jenkins.version>2.46.2</jenkins.version>

```

Comment:
This is most likely not correct. I recommend not providing this property and allowing the default for the parent pom to be used. This would mean a whole lot of users still on 1.x would not be able to use this plugin.